packages/ios_platform_images/ios/ios_platform_images/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import PackageDescription
+
+let package = Package(
+  name: "ios_platform_images",
+  platforms: [
+    .iOS("12.0")
```

Comment:
I removed macOS from the `platforms`. Please let me know if you have concerns with this.